00-HTML-CSS-basics/HTMLExercises/expenses.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+        table, th, td{
+            border: 1px solid black;
+            border-collapse: collapse;
+            margin: auto;
+        }
+        th, td{
+            padding: 10px 50px;
+            text-align: left;
+        }
+        th{
+            background-color: blueviolet;
+            color: white;
+        }
+        td{
+            background-color:powderblue;
+        }
+        body{
+            background-color:steelblue;
+        }
+    </style>
+</head>
+<body>
+    <table>
+        <th>Service</th>
```

Comment:
`<tr>` missing tag, also if you are using `<tbody>` it would be nice to use `<thead>` to markup the header of the table